src/box/tuple_format.h

```diff
@@ -281,6 +293,12 @@ struct tuple_format {
 	struct tuple_constraint *constraint;
 	/** Number of constraints. */
 	uint32_t constraint_count;
+	/**
+	 * Count of consecutive fields of the tuple (starting from field 0) that
+	 * should be checked for null in order to apply a default value.
+	 * In other words, max fieldno with a default value in the format + 1.
+	 */
+	uint32_t defaults_check_count;
```

Comment:
I'd probably name it `default_field_count`. This would be consistent with `index_field_count`, `exact_field_count`, etc, albeit a bit confusing. I'd also put it near other `*_field_count` members.

Reply:
Done.